backend/tournesol/lib/public_dataset.py

```diff
@@ -302,6 +304,7 @@ def write_individual_criteria_scores_file(poll_name: str, write_target) -> None:
             "score": round(criteria_score.score, FLOAT_PRECISION),
             # The voting rights can be very small and reach numbers like 10e-3
             # or even 10e-4. Thus, we round them with more precision.
+            "uncertainty": round(criteria_score.uncertainty, FLOAT_PRECISION),
```

Comment:
The comment above applies to the voting right, and should be displayed before the voting rights, not before uncertainty.

```suggestion
            "uncertainty": round(criteria_score.uncertainty, FLOAT_PRECISION),
            # The voting rights can be very small and reach numbers like 10e-3
            # or even 10e-4. Thus, we round them with more precision.
```